.github/CONTRIBUTING.md

```diff
@@ -11,17 +11,15 @@
 npm install
 
 # build full version of summernote: dist/summernote.js
-npm run build
+npm run build:web
```

Comment:
Is this command is changed with `npm run build:web`?

Reply:
`npm run build:web` is wrong command

i will fix it